astropy/config/paths.py

```diff
@@ -52,14 +59,30 @@ def get_config_dir(rootname: str = "astropy") -> str:
         and not (xchpth := xch / rootname).is_symlink()
     ):
         if xchpth.exists():
-            return str(xchpth.resolve())
+            return xchpth.resolve()
         else:
             linkto = xchpth
 
-    return str(_find_or_create_root_dir("config", linkto, rootname))
+    return _find_or_create_root_dir("config", linkto, rootname)
 
 
-def get_cache_dir(rootname: str = "astropy") -> str:
+def get_config_dir(rootname: str = "astropy") -> str:
```

Comment:
Ah, I see. You added new API instead of changing it. Phew. I might not need downstream testing anymore. Thanks for your thoughtfulness.